README.md

```diff
@@ -59,9 +59,11 @@ $ yarn test:cov
 
 To generate the project documentation use `yarn doc`, it will generate the documentation usually on http://localhost:8080.
 
+Since the app is running, you can access `/api` route to check out the API documentation.
+
 ## TODO
 
-- [ ] Add openapi documentation to get swagger working on a custom endpoint
+- [x] Add openapi documentation to get swagger working on a custom endpoint
```

Comment:
You can remove this item.

Reply:
Resolved on 32735681dffcc15864113ab25766ec7db1b1ad2b.